test/testcafe/spec/DeviceChallengePollViewLoopbackFailfast_spec.js

```diff
@@ -0,0 +1,68 @@
+import { RequestLogger, RequestMock } from 'testcafe';
```

Comment:
I think the tests should be updated in DeviceChallengePollViewOktaVerify_spec.js. And also, the DeviceChallengePollView_spec.js should be updated as well to reflect the case of loopback server login successfully.